instrumentation/github.com/gorilla/mux/otelmux/test/mux_test.go

```diff
@@ -282,3 +285,49 @@ func TestWithPublicEndpointFn(t *testing.T) {
 		})
 	}
 }
+
+type mockConn struct{}
+
+func (m *mockConn) Read(b []byte) (n int, err error)   { return 0, nil }
+func (m *mockConn) Write(b []byte) (n int, err error)  { return 0, nil }
+func (m *mockConn) Close() error                       { return nil }
+func (m *mockConn) LocalAddr() net.Addr                { return nil }
+func (m *mockConn) RemoteAddr() net.Addr               { return nil }
+func (m *mockConn) SetDeadline(t time.Time) error      { return nil }
+func (m *mockConn) SetReadDeadline(t time.Time) error  { return nil }
+func (m *mockConn) SetWriteDeadline(t time.Time) error { return nil }
+
+// Mock recordingResponseWriter for testing purposes.
+type recordingResponseWriter struct {
+	hijacked bool
+	conn     net.Conn
+	rw       *bufio.ReadWriter
+}
+
+func (h *recordingResponseWriter) Hijack() (net.Conn, *bufio.ReadWriter, error) {
+	h.hijacked = true
+	h.conn = &mockConn{}
+	h.rw = bufio.NewReadWriter(bufio.NewReader(h.conn), bufio.NewWriter(h.conn))
+	return h.conn, h.rw, nil
+}
+
+func TestHijack(t *testing.T) {
+	// Arrange
+	h := &recordingResponseWriter{}
+
+	// Act
+	conn, rw, err := h.Hijack()
```

Comment:
This is testing your mock recording response writer, not the one in otelmux. So this doesn't test anything (if you remove the `Hijack()` method from otelmux, the test won't fail.